app/models/user.rb

```diff
@@ -15,4 +15,15 @@ def update_github(user_info)
       github_token: user_info['credentials']['token']
     )
   end
+
+  def bookmark_data
+    ActiveRecord::Base.connection.execute(
+      "SELECT tutorials.title as tutorial_title, tutorials.id as tutorial_id, videos.title as video_title, videos.position as video_position, videos.id AS video_id
+        FROM videos
+        INNER JOIN tutorials ON videos.tutorial_id = tutorials.id
+        INNER JOIN user_videos ON user_videos.video_id = videos.id
+        WHERE user_videos.user_id = #{id}
+        ORDER BY tutorial_id, videos.position"
+    )
+  end
```

Comment:
We might want to refactor using ActiveRecord at some point, but for right now this works well.